### DISCUSSION TITLE
Subscription support

### DISCUSSION THREAD

**Discussion by cmpbedes** [byme8/ZeroQL#88] [General]
Are there any plans for adding support for GraphQL subscriptions?

We've made the transition to this library for all of our mutation/query operations and really like the development experience, but we still have to keep GraphQL.Net around because of the subscriptions

**Comment by byme8** [comment]
I have plans to work on it after adding support for .NET 8 AOT compilation. A lot of refactoring has to happen to make it work. I don't want to do it twice.

**Reply by Mageofdavodo** [reply]
Are there any updates on this? The AOT branch seems to be a bit stale. So any status on when we could expect this? Absolutely love the repo btw 🙌

**Comment by jonmaylandvitrolife** [comment]
Hi @byme8,
I really like your work here. This is exactly what we needed.
Are there any updates regarding subscription support?
